frontend/src/Admin/pages/AssignJudges.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Container } from 'reactstrap'
+import styled from 'styled-components'
+
+import AssignJudgesTable from '../containers/AssignJudgesTable'
+
+const Heading = styled.h1`
+  margin-bottom: 25px;
+`
+
+const ManageJudges = (props) => (
+  <Container fluid>
+    <Heading>Manage Judges</Heading> {/* TODO: Mention the name of the show */}
+    <AssignJudgesTable showId={props.match.params.id} /> {/* TODO: Handle non-number input more gracefully */}
```

Comment:
Can you explain this todo? The show id is an int, what's wrong here?

Reply:
You are correct - the show id will always work because it is an integer, however, `react-router` doesn't limit the type of `id`, so technically you can type into your browser `/admin/show/a/judges` and you would hit the correct page, but Apollo blows up because it doesn't know how to interpret `a` and an id.

I think I have an idea how to fix this by showing the Not Found page if it gets and id it doesn't expect.